templates/starter-mcp-supabase/supabase/functions/mcp-server/deno.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "lib": ["deno.window", "deno.ns"],
+    "strict": true
+  },
+  "imports": {
+    "hono": "npm:hono@^4.6.14",
+    "mcp-lite": "npm:mcp-lite@0.8.2",
+    "zod": "npm:zod@^3.23.8",
```

Comment:
maybe worth using `zod@^4`? Then you can discard the `zod-to-json-schema` converter and use the native `.toJSONSchema()` method: https://zod.dev/json-schema